edenia/antelope-tools
# Refactor/use mui theme 1347

### Refactor to use material theme

### What does this PR do?

- Use SVG format for the logo
- Use the material UI theme to define the palette with the colors and shadows
- Replace colors with theme values
- Overrides the styles of the Paper component instead of duplicate styles for each page
- Remove unused styles and some `!important` rules
- Use primary colors on the Rewards Distribution page

### Steps to test

1. Run the project locally
2. Check the styles are the same

### Screenshots

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/45fbc7ad-774f-49a8-a8f0-d0b79155187e)
